docs/get-started/full-node-deployment.md

```diff
@@ -190,24 +190,40 @@ KillSignal=SIGINT
 TimeoutStopSec=120
 LimitNOFILE=65535
 LimitNPROC=65535
+PrivateTmp=true
+# MemoryMax setting replaces MemoryLimit in newer systemd versions
+MemoryMax=6G
+MemoryHigh=5G
+# keep MemoryLimit for compatibility with older systemd versions
```

Comment:
`Legacy flag, deprecated.` ?

Reply:
no deprecated, centos 8, AWS Linux 2 need this